### PR TITLE
File XML Create Genres

### PR DESCRIPTION
**Pull request by ngochieu1297**
## [Pull request title]
File XML Create Genres 
### [Redmine link]
https://edu-redmine.framgia.vn/issues/7526

### CODE REVIEW

**Comment by ngochieu1297** [comment]
ready

File: app/src/main/res/layout/fragment_genres.xml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+                <RelativeLayout
+                    android:layout_width="match_parent"
+                    android:layout_height="match_parent">
+
+                    <TextView
+                        android:id="@+id/text_music_type"
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content"
+                        android:padding="16dp"
```
Comment (NguyenMinhTuanB) [inline app/src/main/res/layout/fragment_genres.xml]: hardcode

File: app/src/main/res/layout/fragment_genres.xml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+                        android:text="MUSIC TYPE"
```
Comment (NguyenMinhTuanB) [inline app/src/main/res/layout/fragment_genres.xml]: tương tự

File: app/src/main/res/layout/fragment_genres.xml [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+                        android:textAllCaps="true"
+                        android:textColor="@color/color_text_white"
+                        android:textSize="16sp" />
```
Comment (NguyenMinhTuanB) [inline app/src/main/res/layout/fragment_genres.xml]: tương tự

**Comment by ngochieu1297** [comment]
ready

**Pull request merged**